components/AccountOverview/index.tsx

```diff
@@ -175,7 +219,10 @@ const AccountOverview: React.FC<AccountOverviewProps> = ({
           deployer={deployerAddr}
           deployTxHash={account.smart_contract?.deployment_tx_hash}
           udt={account.udt}
-          isVerified={!!account.smart_contract?.abi}
+          address={account.eth_address}
+          isVerified={verifyStatus === 'perfect'}
```

Comment:
Will contracts those verified manually before be marked `is verified`?

Reply:
Not sure, depends on whether it's checked in backend, currently I use result from `sourcify_check_by_addresses` to see if it's verified. If this is not enough to check past verified contracts, I can change to `verifyStatus === 'perfect' || !!account.smart_contract?.abi`

Reply:
> Not sure, depends on whether it's checked in backend, currently I use result from `sourcify_check_by_addresses` to see if it's verified. If this is not enough to check past verified contracts, I can change to `verifyStatus === 'perfect' || !!account.smart_contract?.abi`

`sourcify_check_by_addresses` is not enough because `sourcify` is one of method to verify contracts.

`account.smart_contract.source_code` is the most robust field to check if a contract is verified.

Reply:
Because `source code` will be stored once verified via sourcify